will/plugins/devops/bitbucket_is_up.py

```diff
@@ -4,7 +4,7 @@
 from will.decorators import respond_to, periodic, hear, randomly, route, rendered_template, require_settings
 
 
-class BitBucketIsUpPlugin(WillPlugin):
+class BitbucketIsUpPlugin(WillPlugin):
```

Comment:
Merging this in, but not changing the class name, since it's a potentially breaking change.